dot/rpc/modules/grandpa.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package modules
+
+import (
+	"net/http"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+// GrandpaModule init parameters
+type GrandpaModule struct {
+	blockAPI BlockAPI
+}
+
+// NewGrandpaModule creates a new Grandpa rpc module.
+func NewGrandpaModule(api BlockAPI) *GrandpaModule {
+	return &GrandpaModule{
+		blockAPI: api,
+	}
+}
+
+// ProveFinalityRequest request struct
+type ProveFinalityRequest struct {
+	blockHashStart common.Hash
+	blockHashEnd   common.Hash
+	// authorityID    uint64
```

Comment:
you probably want to uncomment this so that ppl can provide it, but it won't be used internally

Reply:
The linter was having issue with it being an unused variable, wasn't sure how to include it in the present state